<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QtGuiApplicationClass</class>
 <widget class="QMainWindow" name="QtGuiApplicationClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>875</width>
    <height>658</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QtGuiApplication</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <layout class="QVBoxLayout" name="mainVerticalLayout">
      <item>
       <widget class="QListWidget" name="wheatherListWidget"/>
      </item>
      <item>
       <layout class="QHBoxLayout" name="filterHorizontalLayout">
        <item>
         <widget class="QLineEdit" name="filterLineEdit"/>
        </item>
        <item>
         <widget class="QPushButton" name="filterPushButton">
          <property name="text">
           <string>Filter</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="totalHorizontalLayout">
        <item>
         <widget class="QLineEdit" name="totalLineEdit"/>
        </item>
        <item>
         <widget class="QPushButton" name="totalPushButton">
          <property name="text">
           <string>Compute total</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QLineEdit" name="totalMessageLineEdit"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="QtGuiApplication.qrc"/>
 </resources>
 <connections/>
</ui>
